docs/reference/prize-api.md

```diff
@@ -0,0 +1,181 @@
+| chainId                 | the chainId of the network       | 1 for Ethereum mainnet, 137 for Polygon                  |
+| prizeDistributorAddress | address of the prize distributor | the prize distributor address associated with this prize |
+| drawId                  | the integer drawId               | drawId's are unique and sequential over time             |
+
+### All Prizes per Address for a Draw
+
+This API endpoint serves a JSON file for an individual address per Draw, enabling more granular usage/analysis.
+
+`HTTP GET` **https://api.pooltogether.com/prizes/:chainId/:prizeDistributorAddress/draw/:drawId/:address.json**
+
+| Field                   | Description                      | Example/Description                              |
+| ----------------------- | -------------------------------- | ------------------------------------------------ |
+| chainId                 | the chainId of the network       | 1 for Ethereum mainnet, 137 for Polygon          |
+| prizeDistributorAddress | address of the prize distributor | the Prize Distributor associated with this prize |
+| drawId                  | the integer drawId               | drawId's are sequential over time                |
+| address                 | the address of the user          | this address must be lower case                  |
+
+The endpoint will return a `404` status if the address passed was not a winner for that Draw.
+
+#### Examples
+
+For example, using Javascript and [Fetch](https://www.npmjs.com/package/node-fetch), getting the results for drawId 10 on Ethereum mainnet:
+
+```js
+const draw10PrizesResult = await fetch(
+  `https://api.pooltogether.com/prizes/1/0xb9a179dca5a7bf5f8b9e088437b3a85ebb495efe/draw/10/prizes.json`
+);
+const draw10PrizesArray = await draw10PrizesResult.json();
+```
+
+If we wanted to get prizes for every user for drawId's 8 and 9 on Polygon we would make the following requests:
+
+```js
+const urls = [
+  `https://api.pooltogether.com/prizes/137/0x8141bcfbcee654c5de17c4e2b2af26b67f9b9056/draw/8/prizes.json`,
+  `https://api.pooltogether.com/prizes/137/0x8141bcfbcee654c5de17c4e2b2af26b67f9b9056/draw/9/prizes.json`,
+];
+
+const prizes = await Promise.all(
+  urls.map(async (url) => {
+    const resp = await fetch(url);
+    return await resp.json();
+  })
+);
+```
+
+## Local Usage
+
+The data served from this API can also be installed locally using yarn or npm.
+This method may be prefered for data analysis or similar purposes.
+
+#### Example
+
+In your `package.json` add:
+
+```javascript
+  ...
+  "dependencies": {
+    ...
+    "v4PrizesData": "git+https://github.com/pooltogether/v4-draw-results.git
+  }
+  ...
+```
+
+This data can then be imported into scripts for analysis using:
+
+```javascript
+// all winners data for draw 2 on mainet
+import allWinnersDraw15 from "./node_modules/v4PrizesData/api/prizes/1/0xb9a179dca5a7bf5f8b9e088437b3a85ebb495efe/draw/2/prizes";
+```
+
+# Prize Data Lifecycle Overview
```

Comment:
This section doesnt really fit here anymore. Just take the hit and delete it? @asselstine